targets/stm32l432/src/device.c

```diff
@@ -290,7 +290,7 @@ void device_init(int argc, char *argv[])
 
     hw_init(LOW_FREQUENCY);
 
-    if (! tsc_sensor_exists())
+    if (nfc_peripheral_exists())
```

Comment:
I think this if statement and `nfc_peripheral_exists()` can be removed.  So it would just be:

```c
_NFC_status = nfc_init();
```

NFC init already detects if an NFC peripheral is there, and additionally if it's being passively powered.  This allows Solo to become an Tap by simply adding the AS3956 chip.  I'd like the same to remain true for USB-C Solo.  I hadn't thought about future touch sensing Solos when I first wrote that if statement :)

Reply:
Done